main.tf

```diff
@@ -0,0 +1,52 @@
+data "utils_stack_config_yaml" "config" {
+  input                  = [for stack in var.stacks : format("%s/%s.yaml", var.stack_config_local_path, stack)]
+  process_stack_deps     = var.stack_deps_processing_enabled
+  process_component_deps = var.component_deps_processing_enabled
+}
+
+locals {
+  workspaces     = yamldecode(file("${var.workspaces_config_path}"))
+  # variable_sets  = yamldecode(file("${var.variable_set_config_path}"))
+  workspace_tags = { for workspace_name, config in local.workspaces : workspace_name => config.tag_names }
+  workspace_id_map = { for workspace_name, _ in local.workspaces : workspace_name => module.tfc_workspace[workspace_name].id
+  }
+
+  decoded = [for i in data.utils_stack_config_yaml.config.output : yamldecode(i)]
+
+  config = [
+    for stack in local.decoded : {
+      imports = stack.imports,
+      components = {
+        terraform = stack.components.terraform
+      }
+    }
+  ]
+}
+
+module "workspace" {
+  source   = "modules/workspace"
```

Comment:
was not able to find any module in module registry by this path,
https://registry.terraform.io/search/modules?q=modules%2Fworkspace

where you find this?

Reply:
This is local path I just forgot ./ here